### PR TITLE
Documentation: Contact Us

### LINKED ISSUE #42: Documentation: Contact Us section
Add content to the Contact Us section.  This still needs to be determined.  

Ideas:  

- emails to contact
- pointer to logging issue in github
- link to contributing?
- are their other ways in Github to communicate with users?

### PR DESCRIPTION
**Pull request by heycatwonton**
adds link to github discussion and contributing.md on github.

Closes #42

## Type of change

Please delete options that are not relevant.
- [X] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Unit tests
- [X] Integration tests 
- [X] Test coverage >= 90%    <- 91%
- [X] Flake8 Tests 
- [X] Mypy Tests 
- [X] Other - manual, was able to generate documentation successfully with no warning or errors

### Test Configuration
* Python Version: 3.7

## PR Self Evaluation
strikethrough things that don’t make sense for your PR

- ~My code follows the agreed upon best practices~
- ~I have commented my code, particularly in hard-to-understand areas~
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- ~I have added tests or modified existing tests that prove my fix is effective or that my feature works~
- [X] Existing unit tests pass locally with my changes
- ~Any dependent changes have been merged and published in the appropriate modules~
- [X] I have performed a self-review of my own code

### CODE REVIEW

**Review by vchendrix — APPROVED** [review]
@dschristianson This looks good to me.  What do you think?

File: docs/contact.rst [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+| Our preferred channels of communication are public :)
+
+| Do you need help?
+| Do you have ideas?
+| Do you have something to say?
+| Do you have something you want to show us?
+
+`Discuss basin3d
+<https://github.com/BASIN-3D/basin3d/discussions>`_ on GitHub!
```
Comment (vchendrix) [inline docs/contact.rst:12]: @heycatwonton this can be a relative link

File: docs/contact.rst [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+
+
+Caught a bug?
+*************
+If you're interested in contributing to basin3d, check out out our `contributing guidelines <https://github.com/BASIN-3D/basin3d/blob/main/CONTRIBUTING.md>`_. It will help explain why, what, and how to get started.
```
Comment (vchendrix) [inline docs/contact.rst:17]: @heycatwonton don't put the contributing file in a separate PR add it to this pr.  And use a relative reference for the link

**Review by vchendrix — CHANGES_REQUESTED** [review]

**Comment by vchendrix** [comment]
@heycatwonton ok. I merged the contributing placeholder file. I was a little confused at first.  Anyway, in all these markdown docs you should not use absolute urls.  Just relative

File: docs/contact.rst (hunk 2 above)
Comment (heycatwonton) [reply docs/contact.rst]: @vchendrix I made them into relative links, are they correct? Is there a place for the base url? where would that be? For the contributing file, I made it directly on github "add a file" and added it to the project board from the PR, but next time I can make the file on PyCharm instead so it can reference an existing issue.

Comment (vchendrix) [reply docs/contact.rst]: Oops I was thinking that this was a markdown in the code repo.  You should not have a relative link.  Sorry about that.

**Review by vchendrix — APPROVED** [review]
